components/script/dom/htmlinputelement.rs

```diff
@@ -1103,6 +1104,16 @@ impl HTMLInputElement {
     fn selection(&self) -> TextControlSelection<Self> {
         TextControlSelection::new(&self, &self.textinput)
     }
+
+    //Helper function to check if text_cursor has to be updated or not.
+    fn update_text_contents(&self, value: DOMString, update_text_cursor: bool) -> ErrorResult {
+        let output = self.SetValue(value);
+        let mut textinput = self.textinput.borrow_mut();
```

Comment:
Rather than overwriting the current edit point after setting the value, I think it would be more clear to pass the update_text_cursor argument to TextInput::set_content, and have it skip changing the edit point if it's false. This means that the implementation of SetValue will need to move to update_text_contents, rather than update_text_contents calling SetValue. Does that make sense?

Reply:
Ok, so do i delete the setValue function, and copy its implementation to update_text_cursor ? And make the changes to update_text_cursor function accordingly? I did it this way because I was supposed to create a helper function.